### PR TITLE
[disasters] add maps to event pages

### PR DESCRIPTION
**Pull request by beets**
image
image

### CODE REVIEW

File: static/js/apps/event/app.tsx [hunk 1]
```diff
@@ -181,3 +140,74 @@ export function App(props: AppPropsType): JSX.Element {
...
+    ret += startDate;
+    if (endDate) {
+      ret += " — ";
+    }
+  }
+  if (endDate) {
+    ret += endDate;
+  }
+  return ret;
+}
```
Comment (beets) [inline static/js/apps/event/app.tsx:199]: This was a pure move from top of file to here.

File: static/js/apps/event/app.tsx [hunk 2]
```diff
@@ -149,6 +101,13 @@ export function App(props: AppPropsType): JSX.Element {
             </a>
           </h3>
         </div>
+        {(geoJson || location) && (
```
Comment (chejennifer) [inline static/js/apps/event/app.tsx:104]: should this be `|| latLng`?

**Review by chejennifer — COMMENTED** [review]
nice!

File: static/js/apps/event/app.tsx (hunk 2 above)
Comment (beets) [reply static/js/apps/event/app.tsx]: thanks for the catch - done

**Review by chejennifer — APPROVED** [review]

**Pull request merged**